Dockerfile.tpl

```diff
@@ -0,0 +1,3 @@
+FROM #{FROM}
+
+LABEL io.resin.architecture="#{ARCH}"
```

Comment:
I want to have metadata available in all images since our builders will need them. Aleksis will definitely like this.